communities.json

```diff
@@ -672,6 +672,13 @@
       "quote": "Discord both strengthened our community and dramatically improved discussion amongst the developers.",
       "inviteCode": "G7Qnnhy",
       "githubUrl": "https://github.com/nestjs"
+    },
+    {
+      "logo": "ytdl-core.png",
+      "title": "ytdl-core",
+      "quote": "We use Discord to help provide support for the project.",
+      "inviteCode": "nXxFnHR",
+      "githubUrl": "https://github.com/fent/ytdl-core"
```

Comment:
No github URL exists by this name.
Did you mean: https://github.com/fent/node-ytdl-core

Reply:
yes! sorry, it appears i don't know my own urls